src/modules/vtol_att_control/vtol_att_control_main.cpp

```diff
@@ -353,9 +316,12 @@ VtolAttitudeControl::parameters_update_poll()
 	orb_check(_params_sub, &updated);
 
 	if (updated) {
-		struct parameter_update_s param_update;
-		orb_copy(ORB_ID(parameter_update), _params_sub, &param_update);
-		parameters_update();
+		// only update params if disarmed or landed
```

Comment:
This would affect tuning pitch trim, quadchute, perm stab, etc in flight

Reply:
Maybe we should talk about this on the dev call, but I think we need to be more careful about param changes in flight. It's borderline dangerous at the moment for many configurations.

VTOL pitch trim probably doesn't need to exist, and I don't think anything to do with control input or failsafes should be allowed to change while flying.

If we did this across the board then it would be significantly safer to change params for controller tuning while flying.

Reply:
People might want to change quadchute altitude in flight if they takeoff at low altitude. Also, settings these params is the only way a companion can have any control over the behavior.
secondly i think it should be consistent. Most params allow changing in flight, i dont think we should disallow that and i think people tuning an airframe actually use it